conda_gitenv/deploy.py

```diff
@@ -177,12 +151,8 @@ def mocker(self):
                         print('Linking {}/{} to {}'.format(branch.name, label, tag))
                         os.symlink(label_target, label_location)
 
-    conda.install.package_cache_ = orig_package_cache_
-    if isinstance(orig_pkgs_dirs, property):
-        # Support conda>4.1
-        conda.base.context.Context.pkgs_dirs = orig_pkgs_dirs
-    else:
-        conda.install.pkgs_dirs = orig_pkgs_dirs
+    # Undo the monkey patch of the context pkgs_dirs property.
+    conda.base.context.Context.pkgs_dirs = orig_pkgs_dirs
```

Comment:
If we are going to keep doing this, maybe patch as a decorator to the function itself? At least then we are handing exception cases...